src/thor/triplegbuilder.cc

```diff
@@ -1383,6 +1383,10 @@ void TripLegBuilder::Build(
       }
     }
 
+    if (controller.attributes.at(kNodeTransitionTime)) {
+      trip_node->set_transition_time(edge_itr->turn_cost);
```

Comment:
this is where we copy the path edge turn cost into the pbf objects transition_time. turn_cost is probably not good nomenclature... will have to refactor that at some point

Reply:
some of these shouldnt be set, ie when `turn_cost == 0.f`. need to add that

Reply:
ok did that